code/datums/abilities/ability_parent.dm

```diff
@@ -887,9 +887,10 @@
 	proc
 		handleCast(atom/target, params)
 			var/datum/abilityHolder/localholder = src.holder
-			var/result = tryCast(target, params)
 #ifdef NO_COOLDOWNS
-			result = TRUE
+			var/result = TRUE
+#else
+			var/result = tryCast(target, params)
```

Comment:
Looks like this was still accidentally reverted.

Reply:
should be fixed